internal/cli/resume.go

```diff
@@ -0,0 +1,231 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	api "github.com/flightctl/flightctl/api/v1alpha1"
+	apiclient "github.com/flightctl/flightctl/internal/api/client"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+type ResumeOptions struct {
+	GlobalOptions
+	LabelSelector string
+	FieldSelector string
+	All           bool
+}
+
+func DefaultResumeOptions() *ResumeOptions {
+	return &ResumeOptions{
+		GlobalOptions: DefaultGlobalOptions(),
+		LabelSelector: "",
+		FieldSelector: "",
+		All:           false,
+	}
+}
+
+func NewCmdResume() *cobra.Command {
+	o := DefaultResumeOptions()
+	cmd := &cobra.Command{
+		Use:   "resume (device/NAME | device | devices)",
+		Short: "Resume a device or devices based on selectors or all devices.",
+		Long: `Resume devices that are in a conflictPaused state.
+
+Examples:
+  # Resume a specific device
+  flightctl resume device/my-device
```

Comment:
Resume for an individual device which doesn't exist does not return an error and it should return 404.

```
➜ flightctl get dev/f
Error: reading device/f: response status: 404, message: Device of name "f" not found
```

```
❯ flightctl resume dev/f
Resume request for device "f" completed
```

Reply:
As I mentioned in that PR, I think the behaviour is inconsistent with the rest of the CLI commands.
I'd expect an error 404 when the device does not exist vs statusCode 200 when it does exist and it could not be resumed (eg. it already was)